lib/routes/initial_walkthrough/initial_walkthrough.dart

```diff
@@ -254,7 +254,6 @@ class _InitialWalkthroughPageState extends State<InitialWalkthroughPage>
 
   void _restoreFromBackup() async {
     log.info("Restore from Backup");
-    await _signOut();
```

Comment:
Why was this removed?

Reply:
When pressing restore from backup button nothing happens on fresh install. 

Why do we need it in the first place on a fresh install?

Reply:
We want users to be able to switch their Google account without requiring a re-install.

Let's assume I have 2 google accounts: Account A, B
- I selected Google Drive & then Account A, (by mistake)
- Account A's snapshots are loaded or failed to load
- I cancel the flow, press restore again and select Google Drive
- Account A is now selected by default as it's the signed in account.
- User can't switch their Google account on the installed app anymore.